Gemfile.lock

```diff
@@ -304,9 +304,9 @@ GEM
       json
       require_all (~> 3.0)
       ruby-progressbar
-    railties (6.0.3.1)
-      actionpack (= 6.0.3.1)
-      activesupport (= 6.0.3.1)
+    railties (6.0.3.4)
```

Comment:
### [railties](https://rubyonrails.org)

**Hotfix** version upgrade :chart_with_upwards_trend::small_red_triangle: 6.0.3.1 → 6.0.3.4

[_[change-log](https://github.com/rails/rails/blob/v6.0.3.4/railties/CHANGELOG.md), [source-code](https://github.com/rails/rails/tree/v6.0.3.4/railties)_]





<details>
<summary>Commits</summary>

A change of **7** commits. See the full changes on [the compare page](https://github.com/rails/rails/compare/v6.0.3.1...v6.0.3.4).

These are the individual commits:
- (2121b9d) [Only allow ActionableErrors if show_detailed_exceptions is e…](https://github.com/rails/rails/commit/2121b9d20b60ed503aa041ef7b926d331ed79fc2)
- (11052e0) [Update changelog](https://github.com/rails/rails/commit/11052e0091a69d5566624101deb98e4db70e97eb)
- (fbe2433) [Preparing for 6.0.3.2 release](https://github.com/rails/rails/commit/fbe2433be6e052a1acac63c7faf287c52ed3c5ba)
- (4ca2027) [Fix XSS vulnerability in `translate` helper](https://github.com/rails/rails/commit/4ca2027d70ddbab141300e2b8331d2d775c63154)
- (070d4af) [v6.0.3.3](https://github.com/rails/rails/commit/070d4afacd3e9721b7e3a4634e4d026b5fa2c32c)
- (bd5d907) [Prevent XSS in the Actionable Exceptions middleware](https://github.com/rails/rails/commit/bd5d907e72fe799f5455804843a92695a5f29e63)
- (fe76a95) [Preparing for 6.0.3.4 release](https://github.com/rails/rails/commit/fe76a95b0d252a2d7c25e69498b720c96b243ea2)


</details>